kvbc/include/Replica.h

```diff
@@ -228,6 +229,7 @@ class Replica : public IReplica,
   concord::util::ThreadPool blocksIOWorkersPool_;
   std::unique_ptr<concord::client::reconfiguration::ClientReconfigurationEngine> creEngine_;
   std::shared_ptr<concord::client::reconfiguration::IStateClient> creClient_;
+  std::unique_ptr<concord::kvbc::IDbCheckPointManager> dbCheckpointMgr_;
```

Comment:
I think it's better to have an instance of the `concord::kvbc::IDbCheckPointManager` inside the `DBCheckpointManager`. `dbCheckpointMgr_` has no direct use by `ReplicaImp`

Reply:
Fixed